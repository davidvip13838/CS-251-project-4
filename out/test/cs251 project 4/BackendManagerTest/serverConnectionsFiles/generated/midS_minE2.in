500 587
406 195 6708400.2522
262 196 4309722.51677
65 407 9963057.22605
300 37 7339471.57065
79 38 5158653.75074
81 323 2431301.59902
161 476 9064784.65253
394 215 6557442.69071
373 81 263986.58051
327 331 9096427.557
238 226 5051523.25352
300 67 2839198.12098
214 194 1550843.93091
217 145 1985506.44363
373 438 4958985.31595
167 396 4336585.88466
251 408 1972901.28366
270 378 7052403.26986
116 72 6521319.53652
277 268 7613158.14078
125 155 6483735.13526
458 488 5631106.61158
49 28 5829985.04741
25 325 1368484.61582
383 122 9267439.49765
228 343 6236833.77389
84 392 7614642.47121
251 443 1526752.71144
168 353 9220355.21163
430 364 9397749.92759
251 106 6726685.0678
251 229 9772491.20122
67 359 4250944.56966
251 64 1839042.76191
24 434 9780821.26699
339 496 6735754.97718
251 338 391210.47235
251 493 3714356.42035
373 358 5997376.27925
77 10 1761378.65647
63 287 8144444.40516
251 23 1836198.0062
312 8 8708252.91895
353 279 3954806.07755
285 361 6787912.63605
145 253 5016221.92285
410 259 1981218.76454
251 132 6362905.49637
188 462 2056712.11135
311 65 98278.06593
270 290 3084606.05254
111 197 1174141.76853
251 250 2399525.7774
484 444 9624074.79937
406 343 8191367.63596
200 244 4924726.27322
251 208 1058069.44601
280 35 7934208.37504
393 427 8827371.29705
316 372 5069078.2984
457 494 4716967.90153
175 380 2991098.76436
8 264 8912844.0674
81 90 4786647.27214
263 392 727060.41758
35 187 5657508.57819
370 431 8129051.73893
203 393 7678101.19405
482 75 871115.98919
252 227 595351.19579
6 87 704491.12585
51 438 9395135.92884
251 389 9942163.2999
58 112 8843662.27629
165 386 1629728.82194
292 458 687933.31041
230 70 6022669.16143
266 458 7240509.4881
84 447 5135010.15059
6 351 6031112.55164
494 280 948598.21569
174 123 3080087.67067
251 465 5539175.23831
418 251 1739498.0978
128 51 5207084.63902
251 146 1711708.73947
86 366 4062770.68859
369 472 1689476.8504
198 401 5512745.95151
193 148 2225694.17214
65 245 9398606.58472
395 330 7290935.38298
460 404 2128229.84793
54 441 7457446.35651
251 377 2741132.82463
324 81 6561344.93282
249 294 736995.3525
72 435 4814314.51885
187 388 8918888.57977
400 82 3381909.46207
251 470 1601103.19729
247 325 3524919.09497
111 407 3369458.20592
254 265 5771670.45199
118 122 9776549.99872
355 243 4434859.36092
251 171 915915.40887
24 75 1799503.71072
479 96 7239497.57075
294 272 8325596.6907
397 301 4200571.30475
251 403 312221.73829
152 303 4487556.89112
213 446 3314150.26344
137 445 1464483.84117
36 186 5152748.7498
461 71 9479263.12299
272 193 5200361.90351
251 382 6210221.17841
460 361 9053984.31201
251 221 4148972.91116
266 357 4406037.74661
151 10 6759600.47783
394 423 3993034.03243
159 302 510013.24817
487 178 4428537.3594
238 166 9494938.12799
330 123 5030358.84832
386 482 6153167.49274
43 248 4116152.7453
379 69 2656264.66024
251 160 9923568.79757
299 177 5866214.90384
21 143 1897387.24746
37 368 3074885.76083
41 41 7391756.26743
271 234 5020089.65229
372 391 5928285.45881
10 142 5746838.21839
341 235 9651062.53212
251 101 8503093.57167
398 98 3535663.15257
496 273 7875446.70122
55 94 4335352.10309
90 471 2668572.40671
41 469 5642190.99047
371 131 9924719.46864
298 353 5108589.64229
118 358 6433681.66793
128 105 7082478.26951
301 212 5231161.29754
316 364 8024400.11432
1 161 9796938.05995
251 474 1375374.59664
198 327 6238706.0361
251 46 7304580.8336
134 321 1749949.36394
305 354 7004677.84797
321 197 9445024.51321
230 441 414723.01594
251 4 9580163.93101
475 290 8932150.00113
342 151 5551620.21898
156 49 7323810.93138
354 21 6323891.08292
263 446 6243211.64692
219 11 5713299.40406
241 26 1761285.94691
33 96 3175890.756
400 71 8088828.18968
114 353 7619310.93504
335 438 1685591.38551
107 267 8001533.06961
422 11 3994823.23645
243 167 6441952.73969
25 421 285336.62208
308 93 3197914.30989
273 21 7114223.04293
391 273 6076893.38996
487 49 5704680.88474
107 482 596410.60512
259 267 5621199.79648
162 87 8789765.00821
251 138 99702.51955
251 261 1555689.06837
197 118 2518125.0393
173 415 145220.4883
387 315 3853866.3952
456 441 2320881.0629
306 185 4421115.99542
360 353 5645349.05227
251 498 8047701.96068
330 307 1153375.93255
251 337 494703.91102
449 489 4323351.66218
223 20 7404517.13895
282 211 1053995.83724
15 177 4901228.16835
251 183 2140486.42857
66 413 1329401.52795
367 280 1315594.27978
271 418 278739.93437
37 363 981292.33085
56 327 2602320.48219
421 113 6860521.45013
349 442 7804643.93059
203 100 8447149.05279
251 405 9494490.80687
316 394 3408042.50388
35 459 3861350.94823
153 300 9843533.09596
326 190 3676353.94075
42 266 402895.30247
28 29 2209959.11883
251 332 1900671.11485
434 356 6576389.92251
412 136 4975354.88123
455 446 536395.95393
91 388 7278390.21538
422 497 2850020.07559
375 184 638240.71635
296 79 4236684.17487
115 405 1052122.24619
432 459 9029238.52405
236 47 2268344.21477
251 436 308513.7558
179 147 3839128.7929
251 275 2021586.29268
152 431 4457598.83973
274 433 1116092.58599
163 296 8998999.50355
116 43 5251924.55132
251 426 9369761.27364
133 260 5806227.87738
314 352 4850202.50536
460 199 4505370.0852
242 433 1363724.1083
206 198 7162073.29054
399 10 4853247.13715
317 449 1380671.34464
251 202 2663309.7916
251 486 9049874.96992
415 215 3795562.19063
373 363 5557976.38634
5 118 3309008.33238
29 25 2740942.96622
105 88 6848214.12804
35 89 3111694.37906
172 65 13199.46677
7 251 2443933.59383
310 211 3288371.64453
386 271 9619406.38535
251 124 8642508.1726
497 305 4357244.57945
384 119 2605500.47289
251 419 656107.82298
285 102 5242964.00371
53 302 1690830.36799
2 413 2443243.47866
437 91 7625493.36497
354 448 7214023.97887
336 187 8097850.13264
0 16 5216127.98801
288 234 6372577.26291
416 362 9565064.18284
273 122 7873355.9119
66 69 3913734.04182
207 197 9393864.48521
52 475 1799197.40456
251 57 7877816.16388
113 56 1922140.00117
213 351 5473935.35374
68 121 792921.58479
80 335 4684648.0104
440 53 1199988.2021
395 70 3522367.11087
251 289 6236637.37872
44 167 2991814.31306
51 239 4783532.27373
262 407 8320706.82265
401 336 2944288.76729
120 308 3929269.48093
492 136 6999891.13521
351 476 8898473.22412
32 347 5713169.97549
54 251 2807445.47577
263 123 8388253.78279
137 386 7940998.85659
218 259 1941254.949
127 176 3064212.84995
223 175 8555411.42809
449 102 5707703.10918
251 62 5654516.12752
251 329 6364832.09148
114 129 3543041.15126
457 321 8989848.03127
292 211 8838860.37929
192 107 5331728.88672
47 423 4000817.711
3 294 2922944.48381
251 284 6918242.54162
424 174 8882559.95273
355 298 4856145.67709
335 319 5608495.05027
303 294 881113.13256
73 48 5925884.94929
489 190 1393774.56282
163 164 6287002.29455
210 0 6228851.29982
251 334 2994715.15551
384 340 3883705.70878
433 11 9616364.65158
251 473 994493.1301
251 45 3221062.94441
111 82 1084494.29132
394 90 4245926.58864
384 195 1770503.69859
272 173 5650411.25304
20 301 1989103.32988
231 131 141779.96719
26 411 7402546.96968
98 479 2544765.95541
251 121 292513.58243
36 497 1214746.56375
148 139 7149598.86554
251 240 4543143.82941
144 155 9395494.33281
251 95 9470911.16363
0 27 998860.57896
105 52 9981839.09199
365 281 9007690.93254
122 413 7318302.48646
140 268 7011950.56212
251 204 4447892.41052
466 324 471268.39953
30 491 1101176.56286
251 189 7884781.2692
415 299 6762418.4127
307 255 7443035.68257
198 484 5071136.47807
485 306 6127152.70343
88 186 5653562.92302
103 159 2782771.7112
428 477 4435662.7723
341 398 7121075.80319
108 399 6167392.2851
119 63 3185485.68216
325 459 5414916.19943
26 154 564599.035
65 77 2982506.95599
355 18 7737460.71486
251 74 57272.97804
78 335 3141385.08534
188 269 9536027.56466
89 105 335036.12613
99 366 7706456.58288
167 451 5495161.37502
251 59 7540203.04082
167 166 7192993.31437
296 490 1270923.58095
286 54 4155548.28819
39 126 2559761.1136
141 137 6036113.56108
318 364 8236042.79829
92 60 625898.01499
251 281 7749849.71469
251 109 3971971.31777
89 193 2096766.33506
25 257 5004345.12707
117 368 8092530.23663
213 274 5902413.46802
233 148 5629700.02745
254 297 442393.7381
178 379 5242995.09559
295 420 4097813.47964
251 48 2115263.70852
130 246 6774072.61153
114 191 6042906.39044
401 112 9381637.09245
343 440 4661977.87533
444 230 9655401.87697
118 256 446163.0112
251 22 6640138.3518
388 394 2057758.98466
194 464 8519270.88698
222 206 894315.34956
9 259 199047.81296
251 12 4291143.75471
395 355 8380831.38059
6 414 951605.83931
438 398 6450166.95117
123 464 6675332.73782
90 267 5134412.54059
25 111 4247983.39378
442 38 6421673.6526
52 113 7774541.41601
85 141 1224008.83812
352 168 1692412.19334
135 9 255853.26817
141 118 9154333.11977
251 253 2038673.57638
17 168 2003624.09612
251 425 3293497.74085
401 353 2938517.55734
383 154 8313892.37426
499 492 5404593.02337
251 320 9191182.14076
311 478 2902886.8911
352 218 9185026.62928
348 489 6455465.3507
477 192 6659466.53448
322 493 1251157.93678
86 370 3409040.42478
14 463 6826223.88862
277 255 6197717.25575
280 331 4956967.35887
297 185 6366563.66297
251 242 840146.43471
357 178 2599930.74493
198 96 9374917.9189
328 31 8326678.85742
361 363 587990.65458
216 321 2089630.10743
169 18 5118011.18455
126 409 1844241.93653
251 232 821762.08435
82 395 5398261.6529
362 361 5905644.19301
318 469 9750623.30324
251 345 2967430.97624
375 151 9083561.25224
453 403 5717537.25668
399 60 8437084.13753
328 209 69791.20448
110 308 4054077.7787
283 120 5464081.80459
35 407 7646185.37891
336 131 6227881.59274
157 451 5656419.07747
251 258 7345010.13739
98 462 6516662.47029
141 287 6948271.15502
318 134 2645013.74507
230 157 108735.68706
251 376 7779498.05241
44 359 3284648.79388
251 76 3261525.86667
255 239 5757029.13762
207 72 5202369.12175
224 170 3881064.62694
251 350 8244722.13787
21 177 77262.6718
32 227 4708921.28386
418 430 6439497.27723
484 129 5114715.91565
313 488 7285932.39157
43 400 1182401.28175
140 148 3681676.51689
177 467 4641321.30375
42 395 2044734.68914
29 8 9337409.75012
251 180 3994522.53489
75 342 3941528.47987
326 20 8824306.06164
149 363 7694017.81554
291 445 3226684.7826
251 293 8269384.66577
235 197 2389832.6562
237 292 6235505.98369
251 402 2839236.61796
142 123 6925585.04385
81 162 3067720.03207
218 49 5613730.15301
158 62 715230.31571
226 357 7373274.57913
399 304 1853276.89354
499 213 122475.13141
466 34 7938564.67289
455 417 2928581.16322
251 50 8110009.674
251 333 3784240.63341
306 231 8485228.82697
280 78 9410649.3097
251 452 2218593.52645
215 143 2970135.02433
285 451 5639549.70545
251 40 2953274.89353
0 353 2474753.69188
248 336 1688594.40559
314 150 7860320.29338
391 312 9577074.2342
130 187 2196977.01272
251 374 1748281.35264
463 301 618945.40373
251 385 3139098.83764
440 326 5646424.87944
126 41 2002125.83332
356 142 8635801.63792
463 103 886175.23084
167 223 483498.18776
168 294 2129412.56756
477 292 1975074.81935
495 273 5700209.5173
87 260 3736532.52537
108 450 2203726.45961
346 341 3184398.48363
147 295 3137941.27286
449 399 7659165.20834
421 78 2280401.56866
482 108 1885657.10236
255 386 4907660.05825
277 435 1487400.35433
136 472 5893079.06232
201 238 1188157.06886
251 244 5993648.59511
490 273 6722350.3011
343 362 2928543.984
251 83 650417.06275
251 390 6813436.64661
349 65 4333664.29341
219 10 7298474.38954
13 397 7988222.54469
358 235 3284098.40234
224 370 5603817.64151
301 230 5223407.44437
278 252 971162.2518
468 81 9067823.05165
52 375 5464371.56289
179 61 9556252.52068
27 230 7856363.14811
476 347 9385081.93871
251 104 992237.21504
396 343 1384736.67193
26 111 6202386.57521
177 485 1537260.57194
420 9 9243349.96051
153 126 4367796.38366
215 55 6762596.26167
251 491 6646263.16468
478 89 1329228.68913
176 461 8631729.96542
251 182 253298.46438
249 375 7061633.83721
494 401 5120461.92419
276 483 5202065.87156
203 418 1392174.97122
295 415 303187.01552
276 82 8820873.4216
379 481 278886.70568
354 137 2388523.15641
315 239 3857345.71833
311 438 9911307.60203
283 6 1279533.42878
13 348 1725429.74696
352 139 2562527.25886
400 391 3338857.86687
412 344 4185337.70656
341 366 4778832.59187
251 480 6102670.30305
32 448 1467638.47433
19 51 4346183.18598
455 70 7806287.89129
251 454 1250361.74242
472 25 7339846.64464
77 369 4045375.61784
181 144 1956183.06336
497 430 5417356.9831
251 439 9048905.04942
311 197 8271734.57958
251 155 1450461.53961
31 430 4485418.13199
418 490 8055317.35053
251 429 4091278.74606
38 468 4621625.47007
309 237 1442561.39898
97 321 5971951.18785
286 149 7787371.83728
442 225 279944.48811
267 351 1003222.47171
110 326 7755706.77989
251 205 1559092.07981
475 71 6099573.45519
414 220 8016320.63746
99 358 8809852.98234
381 250 8508010.91003
387 174 5768674.9464
256 444 117185.99516
