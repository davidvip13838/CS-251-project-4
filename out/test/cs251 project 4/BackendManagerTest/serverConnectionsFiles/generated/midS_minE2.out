2.1126080045422597E9